recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,5 @@
+libgdal:
```

Comment:
@ocefpaf, the value of this field is available as `%libgdal%`. (Either `2.4` or `3.0`